ScreenControl_Unity/Assets/ScreenControl/ServiceShared/Scripts/HandManager.cs

```diff
@@ -27,6 +27,10 @@ public class HandManager : MonoBehaviour
         public Hand SecondaryHand;
         Chirality secondaryChirality;
 
+        public delegate void HandPresenceEvent();
+        public event HandPresenceEvent HandFound;
+        public event HandPresenceEvent HandsLost;
```

Comment:
As these don't have params, you can remove the delegate and these can be:

'public event Action HandFound'
'public event Action HandsLost'